apps/web/src/pages/tenants/components/UpdateTenantSidebar.tsx

```diff
@@ -82,61 +84,121 @@ export function UpdateTenantSidebar({
-                  ml={-10}
-                />
-              );
-            }}
-          />
-        </Stack>
-      }
-      customFooter={
-        <Group position="apart" w="100%">
-          <Stack spacing={0}>
-            <Text size={'sm'} color={colors.B40}>
-              Updated at {format(new Date(tenant.updatedAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
-            <Text size={'sm'} color={colors.B40}>
-              Created at {format(new Date(tenant.createdAt), 'dd/MM/yyyy HH:mm')}
-            </Text>
+    <>
+      <Sidebar
+        isOpened={isOpened}
+        onClose={onClose}
+        isLoading={isLoadingTenant}
+        onSubmit={(e) => {
+          handleSubmit(onUpdateTenant)(e);
+          e.stopPropagation();
+        }}
+        customHeader={
+          <Stack h={80}>
+            <Group position="apart" spacing={130}>
+              <Controller
+                control={control}
+                name="name"
+                defaultValue=""
+                render={({ field }) => {
+                  return (
+                    <NameInput
+                      {...field}
+                      value={field.value}
+                      data-test-id="tenant-title-name"
+                      placeholder="Enter tenant name"
+                      ml={-10}
+                    />
+                  );
+                }}
+              />
+
+              <Tooltip label={'Delete tenant'} position="bottom">
+                <div
+                  onClick={() => {
+                    setModalIsOpened(true);
+                  }}
+                >
+                  <Trash />
+                </div>
+              </Tooltip>
+            </Group>
           </Stack>
-          <Button
-            disabled={!isDirty || !isValid || isLoadingUpdate}
-            submit
-            loading={isLoadingUpdate}
-            data-test-id="update-tenant-sidebar-submit"
-          >
-            Update
+        }
+        customFooter={
+          <Group position="apart" w="100%">
+            <Stack spacing={0}>
+              <Text size={'sm'} color={colors.B40}>
+                Updated at {format(new Date(tenant.updatedAt), 'dd/MM/yyyy HH:mm')}
+              </Text>
+              <Text size={'sm'} color={colors.B40}>
+                Created at {format(new Date(tenant.createdAt), 'dd/MM/yyyy HH:mm')}
+              </Text>
+            </Stack>
+            <Button
+              disabled={!isDirty || !isValid || isLoadingUpdate}
+              submit
+              loading={isLoadingUpdate}
+              data-test-id="update-tenant-sidebar-submit"
+            >
+              Update
+            </Button>
+          </Group>
+        }
+      >
+        <TenantFormCommonFields control={control} />
+      </Sidebar>
+      <Modal
+        title={
+          <Title size={2} color={colors.warning}>
+            {' '}
+            <WarningIcon /> Delete tenant {tenant.name}?
+          </Title>
+        }
```

Comment:
To be consistent with our other warning modals please use our design system Modal component. You could take a look at a similar modal [here](https://github.com/novuhq/novu/blob/next/apps/web/src/pages/integrations/components/ConditionIconButton.tsx#L80)